src/fsharp/CompileOptions.fs

```diff
@@ -742,12 +742,10 @@ let vsSpecificFlags (tcConfigB: TcConfigBuilder) =
 
 let internalFlags (tcConfigB:TcConfigBuilder) =
   [
-    CompilerOption("use-incremental-build", tagNone, OptionUnit (fun () -> tcConfigB.useIncrementalBuilder <- true), None, None)
```

Comment:
This hidden command line option is never used by the Visual F# Tools (except in one regression test testing a scenario where the use of the option was causing an ICE crash).  It is better to remove it and its implementation since its implementation causes structural problems for the implementation of the improved compiler API in FSharp.Compiler.Service.